frontend/src/app/shared/components/modal-window/modal-window.component.scss

```
@@ -0,0 +1,86 @@
+:host {
+  position: fixed;
+  inset: 0;
+  z-index: 1000;
+
+  display: flex;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)